docker/docker-compose.yml

```diff
@@ -132,7 +133,14 @@ services:
     image: mvertes/alpine-mongo
     container_name: mongo
     healthcheck:
-      test: ["CMD", "mongo", "--quiet", "--eval", "'quit(db.runCommand({ ping: 1 }).ok ? 0 : 1)'"]
+      test:
```

Comment:
is this just formatting?

Reply:
yes, somehow "vscode" does this and I could not disable it. Let me try again.

Reply:
fine with me